### PR TITLE
Ruby 2.6.3 released

### PR DESCRIPTION
**Pull request by nurse**
[x]  Add link to Unicode 12.1 release note
[x] Add links to tarball

### CODE REVIEW

File: en/news/_posts/2019-04-01-ruby-2-6-3-released.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+title: "Ruby 2.6.3 Released"
+author: "naruse"
+translator:
+date: 2019-04-01 00:00:00 +0000
+lang: en
+---
+
+Ruby 2.6.3 has been released.
+
+This release update Unicode version to 12.1, which add to New Japanese Era "〓〓".
```
Comment (sorah) [inline en/news/_posts/2019-04-01-ruby-2-6-3-released.md]: which adds support for

Comment (JuanitoFatas) [reply en/news/_posts/2019-04-01-ruby-2-6-3-released.md]:
```diff
- This release update
+ This release updates
```

File: en/news/_posts/2019-04-01-ruby-2-6-3-released.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+## Download
+
+
+
+
+## Release Comment
+
+Many committers, developers, and users who provided bug reports helped
+us make this release.
```
Comment (JuanitoFatas) [inline en/news/_posts/2019-04-01-ruby-2-6-3-released.md]: Not a big deal, should we merge this with above line? 🙋

File: en/news/_posts/2019-04-01-ruby-2-6-3-released.md (hunk 1 above)
Comment (connorshea) [reply en/news/_posts/2019-04-01-ruby-2-6-3-released.md]: It also should have a `the` before 'Unicode version': 

```suggestion
This release updates the Unicode version to 12.1, which adds support for New Japanese Era "〓〓".
```

File: en/news/_posts/2019-04-01-ruby-2-6-3-released.md (hunk 2 above)
Comment (nurse) [reply en/news/_posts/2019-04-01-ruby-2-6-3-released.md]: meged!

File: en/news/_posts/2019-04-17-ruby-2-6-3-released.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+author: "naruse"
+translator:
+date: 2019-04-017 00:00:00 +0000
+lang: en
+---
+
+Ruby 2.6.3 has been released.
+
+This release adds support for New Japanese Era "令和" (Reiwa).
+It updates the Unicode version to 12.1 beta ([#15195](https://bugs.ruby-lang.org/issues/15195), and updates date library ([#15742](https://bugs.ruby-lang.org/issues/15742).
```
Comment (JuanitoFatas) [inline en/news/_posts/2019-04-17-ruby-2-6-3-released.md]:
```suggestion
It updates the Unicode version to 12.1 beta ([#15195](https://bugs.ruby-lang.org/issues/15195)), and updates date library ([#15742](https://bugs.ruby-lang.org/issues/15742)).
```

It updates the Unicode version to 12.1 beta ([#15195](https://bugs.ruby-lang.org/issues/15195)), and updates date library ([#15742](https://bugs.ruby-lang.org/issues/15742)).

File: ja/news/_posts/2019-04-17-ruby-2-6-3-released.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+author: "naruse"
+translator:
+date: 2019-04-017 00:00:00 +0000
+lang: ja
+---
+
+Ruby 2.6.3 がリリースされました。
+
+このリリースでは日本の新元号「令和」のサポートを追加しています。
+具体的には Unicode のバージョンを 12.1 beta に更新する ([#15195](https://bugs.ruby-lang.org/issues/15195) とともに、date ライブラリを更新しています ([#15742](https://bugs.ruby-lang.org/issues/15742)。
```
Comment (JuanitoFatas) [inline ja/news/_posts/2019-04-17-ruby-2-6-3-released.md]: ditto

```suggestion
具体的には Unicode のバージョンを 12.1 beta に更新する ([#15195](https://bugs.ruby-lang.org/issues/15195)) とともに、date ライブラリを更新しています ([#15742](https://bugs.ruby-lang.org/issues/15742))。
```

具体的には Unicode のバージョンを 12.1 beta に更新する ([#15195](https://bugs.ruby-lang.org/issues/15195)) とともに、date ライブラリを更新しています ([#15742](https://bugs.ruby-lang.org/issues/15742))。

File: en/news/_posts/2019-04-17-ruby-2-6-3-released.md [hunk 5]
```diff
@@ -0,0 +1,48 @@
+---
+layout: news_post
+title: "Ruby 2.6.3 Released"
+author: "naruse"
+translator:
+date: 2019-04-017 00:00:00 +0000
```
Comment (JuanitoFatas) [inline en/news/_posts/2019-04-17-ruby-2-6-3-released.md]:
```suggestion
date: 2019-04-17 00:00:00 +0000
```

File: ja/news/_posts/2019-04-17-ruby-2-6-3-released.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
+---
+layout: news_post
+title: "Ruby 2.6.3 リリース"
+author: "naruse"
+translator:
+date: 2019-04-017 00:00:00 +0000
```
Comment (JuanitoFatas) [inline ja/news/_posts/2019-04-17-ruby-2-6-3-released.md]:
```suggestion
date: 2019-04-17 00:00:00 +0000
```

**Pull request merged**